canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -234,6 +237,8 @@ class RadiosetControl extends React.Component {
 					orientation={this.props.control.orientation}
 					helperText={this.props.control.helperText}
 					readOnly={this.props.readOnly}
+					// Provide unique aria-label for each radio button group
+					aria-label={`${radioBtnGroupAriaLabel} (${ControlUtils.getDataId(this.props.control, this.props.propertyId)})`}
```

Comment:
Seems odd have the dataId in the text.  Is there a reason for that?

Reply:
Since `aria-label` should be unique for each radio group I have provided `dataId`. Without `dataId` assessment tool is throwing below error. Is there any alternative that we can use instead of dataId ?

<img width="1164" alt="Screenshot 2025-01-28 at 12 12 22 PM" src="https://github.com/user-attachments/assets/60cf6297-f196-496d-8fe2-bd62ffdf7cd0" />